package.json

```diff
@@ -31,21 +34,18 @@
     "css-loader": "~0.23.1",
     "del": "~2.2.0",
     "edx-pattern-library": "~0.12.5",
+    "eslint-config-edx": "bjacobel/eslint-config-edx#bjacobel/feedback",
```

Comment:
The `0.0.1` that's on NPM right now was released last Friday so it's a little stale, but I'm going to release a `1.0.0` to NPM after https://github.com/edx/eslint-config-edx/pull/1 merges and then this can be updated to point there.